package.json

```diff
@@ -5,8 +5,9 @@
   "private": true,
   "devDependencies": {
     "enzyme": "^2.4.1",
+    "jsdom": "^9.6.0",
```

Comment:
Since enzyme's `mount` API requires a DOM, JSDOM is required in order to use mount if you are not already in a browser environment (ie, a Node environment).

For a best experience with enzyme, it is recommended to load a document into the global scope before requiring React for the first time.

http://airbnb.io/enzyme/docs/guides/jsdom.html